src/runtime/mod.rs

```diff
@@ -24,6 +24,15 @@ pub struct Runtime {
 
     /// Tokio runtime, always current-thread
     rt: tokio::runtime::Runtime,
+
+    /// This is here for drop order reasons.
+    ///
+    /// We can't unset the driver in the runtime drop method, because the inner runtime needs to
```

Comment:
What is the 'inner' runtime referred to here?

Reply:
The tokio runtime we store.